### PR TITLE
vocal: fix build

### PR DESCRIPTION
**Pull request by moni-dz**
###### Motivation for this change
https://hydra.nixos.org/build/144464989

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 125386` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>vocal</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 125386` at 07e6d1c1 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>vocal</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/applications/audio/vocal/default.nix:91:5:
```
   |
91 |     maintainers = with maintainers; [ ];
   |     ^
```
</li>
  </ul>
</details>

**Review by siraben — APPROVED** [review]
LGTM

File: pkgs/applications/audio/vocal/default.nix [hunk 1]
```diff
@@ -61,13 +63,24 @@ stdenv.mkDerivation rec {
     glib-networking
   ];
 
+  /*
+    granite 6.0.0 removed about dialogs
+    see: https://github.com/needle-and-thread/vocal/issues/483
+  */
+  patches = [
+    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/vocal/default.nix:71]:
```suggestion
  patches = [
    # granite 6.0.0 removed about dialogs
    # see: https://github.com/needle-and-thread/vocal/issues/483
    (fetchpatch {
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**